__tests__/selection.js

```diff
@@ -148,6 +180,37 @@ describe('selection', () => {
     });
   });
 
+  describe('findParentNodeOfTypeClosestToPos', () => {
+    it('should find parent node of a given `nodeType` if a given `$pos` is directly inside it', () => {
+      const { state } = createEditor(doc(p('hello')));
+      const { paragraph } = state.schema.nodes;
+      const { node } = findParentNodeOfTypeClosestToPos(
+        state.doc.resolve(2),
+        paragraph
+      );
+      expect(node.type.name).toEqual('paragraph');
+    });
```

Comment:
These are kind of duplicate test right? Same scenario was already covered by `findParentNodeClosestToPos`.